src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -6,19 +6,54 @@
 import { Action2, registerAction2 } from 'vs/platform/actions/common/actions';
 import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 import { KeybindingWeight } from 'vs/platform/keybinding/common/keybindingsRegistry';
-import { TerminalLocation } from 'vs/platform/terminal/common/terminal';
+import { TerminalLocation, terminalTabFocusContextKey } from 'vs/platform/terminal/common/terminal';
+import { editorTabFocusContextKey } from 'vs/workbench/browser/parts/editor/tabFocus';
 import { AccessibilityHelpWidget } from 'vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibilityHelp';
-import { ITerminalEditorService, ITerminalGroupService, ITerminalInstance, ITerminalService } from 'vs/workbench/contrib/terminal/browser/terminal';
-import { TerminalCommandId } from 'vs/workbench/contrib/terminal/common/terminal';
+import { ITerminalContribution, ITerminalEditorService, ITerminalGroupService, ITerminalInstance, ITerminalService, IXtermTerminal } from 'vs/workbench/contrib/terminal/browser/terminal';
+import { ITerminalProcessManager, TerminalCommandId } from 'vs/workbench/contrib/terminal/common/terminal';
 import { TerminalContextKeys } from 'vs/workbench/contrib/terminal/common/terminalContextKey';
 import { terminalStrings } from 'vs/workbench/contrib/terminal/common/terminalStrings';
+import { AccessibleBufferWidget } from 'vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer';
+import { registerTerminalContribution } from 'vs/workbench/contrib/terminal/browser/terminalExtensions';
+import { DisposableStore } from 'vs/base/common/lifecycle';
+import { Terminal } from 'xterm';
+import { TerminalWidgetManager } from 'vs/workbench/contrib/terminal/browser/widgets/widgetManager';
 
 const category = terminalStrings.actionCategory;
 
+class AccessibleBufferContribution extends DisposableStore implements ITerminalContribution {
+	static readonly ID: 'terminal.accessible-buffer';
+	static get(instance: ITerminalInstance): AccessibleBufferContribution | null {
+		return instance.getContribution<AccessibleBufferContribution>(AccessibleBufferContribution.ID);
+	}
+
+	private _accessibleBufferWidget: AccessibleBufferWidget | undefined;
+	get accessibleBufferWidget(): AccessibleBufferWidget | undefined { return this._accessibleBufferWidget; }
+
+	constructor(
+		instance: ITerminalInstance,
+		private readonly _processManager: ITerminalProcessManager,
+		private readonly _widgetManager: TerminalWidgetManager,
+		@IInstantiationService private readonly _instantiationService: IInstantiationService,
+	) {
+		super();
+	}
+
+	xtermReady(xterm: IXtermTerminal & { raw: Terminal }): void {
+		this._accessibleBufferWidget = this._instantiationService.createInstance(AccessibleBufferWidget, xterm, this._processManager, this._widgetManager);
```

Comment:
You can avoid passing in widget manager and instead of `_processManager` you could pass `capabilities`?